api/bazel/repository_locations.bzl

```diff
@@ -100,9 +100,9 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "OpenTelemetry Proto",
         project_desc = "Language Independent Interface Types For OpenTelemetry",
         project_url = "https://github.com/open-telemetry/opentelemetry-proto",
-        version = "0.11.0",
-        sha256 = "985367f8905e91018e636cbf0d83ab3f834b665c4f5899a27d10cae9657710e2",
-        release_date = "2021-10-07",
+        version = "0.16.0",
+        sha256 = "22763df2020d4e16a411b249f86dda8fa1bea69a9592915bf0b1dab6d7005190",
+        release_date = "2022-04-01",
```

Comment:
`0.18.0` is now out - https://github.com/envoyproxy/envoy/issues/21351

Reply:
Updated, thanks!